### ISSUE TITLE
SVN in manual

### ISSUE THREAD

**Issue by maristu** [filipealtoe/LabVIEWforRasPiVIPM#2]
It's a minnor issue. To install svn into the raspberry the command I had to use was:

sudo apt-get install subversion

**Comment by filipealtoe** [comment]
Yes; there is a typo on the document that is now fixed for the next Beta release. 

Thanks for reporting it. 
Filipe
